src/lib/isMACAddress.js

```diff
@@ -1,19 +1,18 @@
 import assertString from './util/assertString';
 
-const macAddress = /^([0-9a-fA-F][0-9a-fA-F]:){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressNoColons = /^([0-9a-fA-F]){12}$/;
-const macAddressWithHyphen = /^([0-9a-fA-F][0-9a-fA-F]-){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressWithSpaces = /^([0-9a-fA-F][0-9a-fA-F]\s){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressWithDots = /^([0-9a-fA-F]{4}).([0-9a-fA-F]{4}).([0-9a-fA-F]{4})$/;
+const macAddress = /^(?:[0-9a-fA-F]{2}([-:\s]))([0-9a-fA-F]{2}\1){4}([0-9a-fA-F]{2})$/;
+const macAddressNoSeparators = /^([0-9a-fA-F]){12}$/;
```

Comment:
I think `noSeparator` fits better with the aim of the options since we are excluding every kind of separator and not only colons.

Reply:
I agree with you, can we make it snake case to follow the naming convention of other validators? and i guess we will  need to remove `no_colons` support in the next major version